src/lib/components/workflow/workflows-summary-configurable-table/table-header-row.svelte

```diff
@@ -31,48 +29,39 @@
   $: label = translate('workflows', 'select-all-workflows');
 </script>
 
-<tr class="workflows-summary-configurable-table-header-row" class:pinned>
-  {#if pinned && $supportsBulkActions}
+<tr>
+  {#if !empty && $supportsBulkActions}
     <th class="batch-actions-checkbox-table-cell">
-      {#if !empty}
-        <Checkbox
-          {label}
-          labelHidden
-          id="select-visible-workflows"
-          data-testid="batch-actions-checkbox"
-          aria-label={label}
-          onDark
-          hoverable
-          bind:checked={$pageSelected}
-          {indeterminate}
-          on:change={handleCheckboxChange}
-        />
-      {/if}
+      <Checkbox
+        {label}
+        labelHidden
+        id="select-visible-workflows"
+        data-testid="batch-actions-checkbox"
+        onDark
+        hoverable
+        bind:checked={$pageSelected}
+        {indeterminate}
+        on:change={handleCheckboxChange}
+      />
     </th>
   {/if}
-  {#if pinned && $supportsBulkActions && $batchActionsVisible}
-    <th class="batch-actions-table-cell" colspan={pinnedColumns.length || 1}>
+  {#if $supportsBulkActions && $batchActionsVisible}
+    <th class="batch-actions-table-cell" colspan={columnsCount}>
       <BatchActions {workflows} />
     </th>
   {:else}
     <slot />
   {/if}
-  {#if !pinned}
-    <th class="configuration-button-table-cell">
-      <IconButton
-        data-testid="workflows-summary-table-configuration-button"
-        icon="vertical-ellipsis"
-        on:click={onClickConfigure}
-      />
-    </th>
-  {/if}
+  <th class="configuration-button-table-cell">
+    <IconButton
```

Comment:
Looks like this starts floating left a bit with fewer columns

<img width="1600" alt="Screenshot 2023-08-02 at 12 04 45 PM" src="https://github.com/temporalio/ui/assets/15069288/fa8e862a-43f4-445f-8d7b-3b6c72931ec2">